### PR TITLE
Added Senior Engineering Manager Position and focus roles

### PR DESCRIPTION
**Pull request by DCTFUK**
- Added senior-engineering-manager.md to levels (should this be renamed positions).
- Added role for each focus (x3)
- Updated readme.md 
- Fixed typo in engineering-manager.md

### CODE REVIEW

File: levels/roles/sem_consulting.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+### Delivery Synergy
+
+- elaborates with the **Delivery Function** on time and resources for the
+  opportunity
+- resourcing _capacity_ and _capability_
+- team formation as a framework
+- early assessment on required skills and levels
+- monitoring and alerting
+- technical refinement and evaluation of proposed solution (SoW)
+- resourcing Capability (skills and levels)
```
Comment (GooseLF) [inline levels/roles/sem_consulting.md:41]: Nitpicking here but we move 🙌

Lowercase "c"?

File: levels/roles/sem_people.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Senior Engineering Manager - People focus
+
+The 'SEM - People focus' role is concerned with effective and efficient
+recruitment, continuous and professional development and retention of Data
+Engineers and Technical Operations Engineers at The Data Shed. This is achieved
+through the following processes:
+
+## Recruitment
+
+- working closesly with the Talent Aquisition Manager to ensure high quality
```
Comment (GooseLF) [inline levels/roles/sem_people.md:10]: *closely

File: levels/roles/sem_people.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  standards around the recruitment of Data Engineers.
+- ensuring that key skills are recruited for where required.
+
+## Development
+
+- ensuring that Data Engineers are engaged in a stuiable Learning and
+  development plan and have approparite objectives related to it.
+- ensuring Learning and Development days are appropriately planned and utilized.
+- identitfying skill gaps and appropriate Champions that can help reduce those
+  gaps through mentoring and knowledge sharing (utilzing the skills matrix).
```
Comment (GooseLF) [inline levels/roles/sem_people.md:20]: *utilising (UK Spelling)
*utilizing (US Spelling)

Whichever you prefer most 🥸

File: guides/acronyms.md [hunk 4]
```diff
@@ -16,6 +16,8 @@
 
 *[PM]: Project Manager
 
+*[SEM]: Senior Engineering Manager
+
```
Comment (GooseLF) [inline guides/acronyms.md:20]: Should "SFIA" be added to this? Apologies if it's defined elsewhere but I'm seeing it in these pages but not sure what it means? 🤯

**Review by GooseLF — CHANGES_REQUESTED** [review]
Hope I'm not overstepping the mark with this considering I've been here 3 days.

Also, can we squash the commits pls? ^_^

File: levels/roles/sem_people.md (hunk 3 above)
Comment (GooseLF) [reply levels/roles/sem_people.md:20]: Touche, regardless, it's still missing an "i" 😉🤣

File: levels/roles/sem_consulting.md (hunk 1 above)
Comment (DCTFUK) [reply levels/roles/sem_consulting.md]: yeah will squash commits going forward.

File: levels/roles/sem_consulting.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+### Engineering Consulting - Tech Ops Synergy
+
+- potential Tech Ops/Service Managemnt opportunities at early and mid delivery
+  stage
+- account growth opportunity
+
+### Engineering Consulting - People Synergy
+
+- resourcing Capability
```
[the remark this answers is not in the document]
Comment (DCTFUK) [reply levels/roles/sem_consulting.md]: resolved

File: levels/roles/sem_people.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+## Recruitment
+
+- working closesly with the Talent Aquisition Manager to ensure high quality
+  standards around the recruitment of Data Engineers.
+- ensuring that key skills are recruited for where required.
+
+## Development
+
+- ensuring that Data Engineers are engaged in a stuiable Learning and
```
[the remark this answers is not in the document]
Comment (DCTFUK) [reply levels/roles/sem_people.md]: resolved

File: guides/acronyms.md (hunk 4 above)
Comment (DCTFUK) [reply guides/acronyms.md:20]: Anything like that should be added yeah. Can put it in a different branch though. 

https://sfia-online.org/en/sfia-8/sfia-8

File: levels/roles/sem_people.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  development plan and have approparite objectives related to it.
```
Comment (GooseLF) [inline levels/roles/sem_people.md:17]: You're going to hate me 🤣 *appropriate

File: levels/roles/sem_people.md (hunk 3 above)
Comment (GooseLF) [inline levels/roles/sem_people.md:20]: Not sure if you saw my comment on this before it got resolved so just re-iterating. Still missing "i"

File: guides/acronyms.md (hunk 4 above)
Comment (GooseLF) [reply guides/acronyms.md:20]: Sound, created issue #66 for this.

**Comment by GooseLF** [comment]
Pushed a commit fixing some spelling errors

**Review by GooseLF — APPROVED** [review]

**Pull request merged**